pkg/render/logstorage.go

```diff
@@ -411,9 +410,7 @@ func (es *elasticsearchComponent) Objects() ([]client.Object, []client.Object) {
 	} else {
 		toCreate = append(toCreate,
 			createNamespace(ElasticsearchNamespace, es.installation.KubernetesProvider),
-			createNamespace(KibanaNamespace, es.installation.KubernetesProvider),
```

Comment:
What's creating the kibana namespace now? Was this a mistake to remove this?

Reply:
This code path is for Managed clusters only and they do not need the kibana namespace or service any longer since there are no components in Managed clusters that make calls to Kibana. Please let me know if that's not the case but I'm fairly certain that it is.

Reply:
No you're right, I just didn't realise this was for a managed cluster. Thanks for cleaning that up.